<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LNDA0021" name="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality" tests="11" failures="0" timestamp="9 Nov 2015 11:21:50 GMT" time="119.792" errors="8">
  <testcase name="Step_06_verify_Hide_Edit_And_Delete" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_06_verify_Hide_Edit_And_Delete()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_06_verify_Hide_Edit_And_Delete()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_06_verify_Hide_Edit_And_Delete -->
  <testcase name="Step_11_verify_Hide_Action" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_11_verify_Hide_Action()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_11_verify_Hide_Action()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_11_verify_Hide_Action -->
  <testcase name="Step_09_verify_Hide_Edit_And_Delete" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_09_verify_Hide_Edit_And_Delete()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_09_verify_Hide_Edit_And_Delete()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_09_verify_Hide_Edit_And_Delete -->
  <testcase name="Step_07_Verify_ShowMore_OR_Less" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_07_Verify_ShowMore_OR_Less()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_07_Verify_ShowMore_OR_Less()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_07_Verify_ShowMore_OR_Less -->
  <testcase name="Step_08_verify_Flag_And_Points" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_08_verify_Flag_And_Points()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_08_verify_Flag_And_Points()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_08_verify_Flag_And_Points -->
  <testcase name="Step_12_verify_Toogle_Button_state_Action" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_12_verify_Toogle_Button_state_Action()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_12_verify_Toogle_Button_state_Action()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_12_verify_Toogle_Button_state_Action -->
  <testcase name="Step_05_Verify_RollingWeekUI_And_Select_TopicView" time="0.000" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.Throwable" message="Method TC01_Instructor_Verify_Week_View_Functionality.Step_05_Verify_RollingWeekUI_And_Select_TopicView()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC01_Instructor_Verify_Week_View_Functionality.Step_05_Verify_RollingWeekUI_And_Select_TopicView()[pri:0, instance:com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_05_Verify_RollingWeekUI_And_Select_TopicView -->
  <testcase name="Step_03_Verify_RollingWeekUI" time="32.283" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality"/>
  <testcase name="Step_01_Instructor_Logs_in_to_the_Application" time="30.291" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality"/>
  <testcase name="Step_02_Instructor_Manages_Course_From_SSO" time="15.064" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality"/>
  <testcase name="Step_04_Verify_OfflineGraded_Activities_Displays" time="42.154" classname="com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:302)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$100(ExpectedConditions.java:41)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:288)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:285)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at com.qait.mindtap.automation.utils.SeleniumWait.waitForElementToBeVisible(SeleniumWait.java:53)
	at com.qait.mindtap.automation.getpageobjects.GetPage.isElementDisplayed(GetPage.java:208)
	at com.qait.mindtap.keywords.OfflineActivityAction.instructorEntersAssociatedTopic(OfflineActivityAction.java:64)
	at com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality.Step_04_Verify_OfflineGraded_Activities_Displays(TC01_Instructor_Verify_Week_View_Functionality.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- Step_04_Verify_OfflineGraded_Activities_Displays -->
</testsuite> <!-- com.qait.mindtap.tests.NG_30457.TC01_Instructor_Verify_Week_View_Functionality -->
